interdiff.go

```diff
@@ -186,6 +200,60 @@ func ApplyDiff(source string, diffFile *diff.FileDiff) (string, error) {
 	return strings.Join(newBody, "\n"), nil
 }
 
+// convertChunksIntoFileDiff form hunks of fileDiff from array of chunks
+func convertChunksIntoFileDiff(chunks []diffFinder.Chunk, fileDiff *diff.FileDiff) {
+	var currentOldI, currentNewI int32 = 1, 1
+	currentHunk := diff.Hunk{OrigStartLine: currentOldI, NewStartLine: currentNewI}
+	var currentHunkBody []string
+	for _, c := range chunks {
+		for _, line := range c.Added {
+			currentHunkBody = append(currentHunkBody, "+"+line)
+			currentNewI++
+		}
+		for _, line := range c.Deleted {
+			currentHunkBody = append(currentHunkBody, "-"+line)
+			currentOldI++
+		}
+
+		if len(c.Equal) > minUnchangedHunkSize {
+			if len(currentHunkBody) > 0 {
+				for _, line := range c.Equal[:indentSize] {
+					currentHunkBody = append(currentHunkBody, " "+line)
+				}
+				currentHunk.OrigLines = currentOldI + indentSize + 1 - currentHunk.OrigStartLine
+				currentHunk.NewLines = currentNewI + indentSize + 1 - currentHunk.NewStartLine
+				currentHunk.Body = []byte(strings.Join(currentHunkBody, "\n"))
+				fileDiff.Hunks = append(fileDiff.Hunks, &currentHunk)
+			}
+
+			currentOldI += int32(len(c.Equal))
+			currentNewI += int32(len(c.Equal))
+
+			currentHunk = diff.Hunk{OrigStartLine: currentOldI - indentSize, NewStartLine: currentNewI - indentSize}
+
+			// Clean currentHunkBody
+			currentHunkBody = currentHunkBody[:0]
```

Comment:
```suggestion
			currentHunkBody = []string{}
```

that is a more clear reset of the value.